Movies-MVP/Movies-MVP/Core/Router/Router.swift

```diff
@@ -0,0 +1,38 @@
+// Router.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Router
```

Comment:
```suggestion
/// App routing 
```
Например зачем мне дубли в доке?